pygitguardian/client.py

```diff
@@ -357,6 +357,7 @@ def content_scan(
         self,
         document: str,
         filename: Optional[str] = None,
+        all_secrets: Optional[bool] = None,
```

Comment:
Can you move the new argument at the end? If we don't then we break the API if the function is passed a value for `extra_headers` without using a keyword argument for it.

Actually it would be a good idea to add a `*,` after `extra_headers` so that we don't have this issue with future arguments.

Reply:
Done

Reply:
What about `multi_content_scan` ? Should I make `ignore_known_secrets` mandatory kwarg ?

Reply:
Made it like that for now, so as not to introduce breaking changes:

```python
        ignore_known_secrets: Optional[bool] = None,
        *,
        all_secrets: Optional[bool] = None,
```